source/Octo.Tests/Commands/CreateReleaseCommandFixture.cs

```diff
@@ -60,12 +63,11 @@ public void ShouldThrowForBadTenant()
             CommandLineArgs.Add("--apikey=API-test");
             CommandLineArgs.Add("--project=Test Project");
             CommandLineArgs.Add("--releaseNumber=1.0.0");
-            CommandLineArgs.Add("--tenant=bad");
+            CommandLineArgs.Add("--tenant=badTenant");
             CommandLineArgs.Add($"--deployto={ValidEnvironment}");
-            Assert.ThrowsAsync<CommandException>(async delegate
-            {
-                await createReleaseCommand.Execute(CommandLineArgs.ToArray());
-            });
+
+            var ex = Assert.ThrowsAsync<CouldNotFindException>(() => createReleaseCommand.Execute(CommandLineArgs.ToArray()));
+            ex.Message.Should().Be("The tenant 'badTenant' does not exist or the account does not have access.");
```

Comment:
Slight tweak to the wording
```suggestion
            ex.Message.Should().Be("The tenant 'badTenant' does not exist or you do not have permissions to view it.");
```
the `the account` part could be unclear to some users
(If you change this, dont forget to change everywhere)

Reply:
Done.